.devcontainer/devcontainer.json

```diff
@@ -69,6 +69,17 @@
   "updateContentCommand": "bin/vscode-setup",
   "postCreateCommand": "echo 'source /usr/share/bash-completion/completions/git' >> ~/.bashrc",
 
-  // Use 'forwardPorts' to make a list of ports inside the container available locally.
-  "forwardPorts": [3390, 5432, 9000, 9999]
+  // Use 'forwardPorts' to make a list of ports inside the container available locally. The ports correspond to the following:
```

Comment:
Is this sentence stray? Not sure it makes sense with the new format?

Reply:
Yep you're right - thanks for catching!